gtsam.h

```diff
@@ -814,6 +814,7 @@ class Unit3 {
   // Other functionality
   Matrix basis() const;
   Matrix skew() const;
+  gtsam::Point3 point3() const;
```

Comment:
as_point?

Reply:
Wouldn't that break backwards compatibility?

Reply:
@ProfFan gtsam.h is just a wrapper. Maybe the original function is badly named, but that's orthogonal to this PR. And, indeed, would be an API change.